### ISSUE TITLE
Error 126 -- Problem loading dll !

### ISSUE THREAD

**Issue by navstar008** [dougforpres/SonyCamera#11]
C:\Program Files\Common Files\ASCOM\Camera>sonycamerainfo
Sony Camera Info
~~~~~~~~~~~~~~~~

Loaded 'turbojpeg.dll' from 'C:\WINDOWS\SYSTEM32\turbojpeg.dll'
Unable to load 'libraw.dll' - error 126Problem loading dlls

----

-----
Currently trying V 1.0.0.31.
Will try with older versions.
Thanks for the great work!

**Comment by dougforpres** [comment]
Sorry for delayed response - I had _thought_ I had disabled bug reporting here on Github - you see, Github will not notify me when someone creates or updates an issue (for "reasons" it doesn't seem to let the "owner" be notified... and when the owner is the only person working on the code that can be problematic).

Error 126 is a code that windows throws out whenever it cannot load a DLL.
Windows doesn't give any more info - though at least you _know_ which dll is having the trouble.

It is possible you're missing some runtime DLL's.
Here is a list of DLL's that the libraw.dll loads (I dumped a list of loaded DLL's before and after loading libraw)

C:\WINDOWS\System32\WS2_32.dll  Microsoft« Windows« Operating System    10.0.19041.1081
C:\WINDOWS\System32\RPCRT4.dll  Microsoft« Windows« Operating System    10.0.19041.1
C:\WINDOWS\SYSTEM32\MSVCP120.dll        Microsoft« Visual Studio« 2013  12.00.40660.0
C:\WINDOWS\SYSTEM32\MSVCR120.dll        Microsoft« Visual Studio« 2013  12.00.40660.0

Try following the instructions here to download and install the older VS 2013 runtime: https://github.com/dougforpres/ASCOMSonyCameraDriver/wiki/Troubleshooting#driver-wont-load
